### ISSUE TITLE
ComboBox component DAP issue "Id references used in WAI-ARIA properties must be valid"

### ISSUE THREAD

**Issue by Kandianne** [carbon-design-system/carbon#6597]
Versions of Carbon:
"carbon-components": "10.11.1",
"carbon-components-react": "7.11.1"

## Detailed description
When using the ComboBox Component within a Table, the following DAP violation occurs when I run the scan "Id references used in WAI-ARIA properties must be valid"
![image](https://user-images.githubusercontent.com/12722441/88961793-82750300-d273-11ea-89e6-29576feae78a.png)

## Steps to reproduce the issue

Example: https://codesandbox.io/s/optimistic-cookies-2gv9y?file=/src/index.js
1. Run DAP in the above example

**Comment by joshblack** [comment]
Hey there @Kandianne! 👋 

I believe the challenge here is coming from the component not being supplied `titleText`. As a result, the label for the combobox isn't being rendered which is what that warning refers to.

@dakahn for these sorts of inline situations for dropdowns, what is the preferred path forward? Do they always need a visible label or is this something we can configure on the downshift end?

**Comment by dakahn** [comment]
[WCAG Success Criterion 3.3.2](https://www.w3.org/TR/WCAG21/#labels-or-instructions) requires that form inputs have accessible labeling for both sighted users _and_ users of assistive technologies. For a deeper dive [see the section **Intent** in this Labeling and Instructions guide](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html),

**Comment by tw15egan** [comment]
Seems related to https://github.com/carbon-design-system/carbon/issues/6701

https://github.com/carbon-design-system/carbon/issues/6701#issuecomment-684005971

**Comment by dakahn** [comment]
Closing as resolved. Can reopen if I missed something.
